package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "studio-ui",
     "private": true,
-    "version": "1.10.8",
+    "version": "1.11.0-rc.0",
```

Comment:
this will break existing behaviour, please just use 11.1 we will check later how to resolve it